### ISSUE TITLE
Duplicated strings

### ISSUE THREAD

**Issue by bmarty** [vector-im/element-x-android#1356]
The is an issue on Strings between iOS and Android application.

On Android, some strings have been added for submit buttons, like for instance:

https://github.com/vector-im/element-x-android/blob/develop/features/login/impl/src/main/res/values/localazy.xml#L40
```xml
<string name="screen_account_provider_continue">"Continue"</string>
```

instead of using the generic string
https://github.com/vector-im/element-x-android/blob/develop/libraries/ui-strings/src/main/res/values/localazy.xml#L19
```xml
<string name="action_continue">"Continue"</string>
```

The value is the same in English, but translations can differ (maybe not for this particular example though), which is not what we want.

We should simplify by removing the extra string entries, and use the generic value when it exists. It will reduce the number of strings to translate and will ensure that Android and iOS are synced. The drawback is that it will not be possible to tweak a value for a particular string by just updating Localazy.

We can use the find duplicate facilities on Localazy to detect identical strings, and probably remove all the duplicates.

**Comment by bmarty** [comment]
Other string to remove:
- `screen_room_retry_send_menu_remove_action` (we now use `action_remove`)

**Comment by pixlwave** [comment]
Updating in EXI: https://github.com/vector-im/element-x-ios/pull/1827

`common_select_your_server` → `screen_change_server_title`
`screen_room_retry_send_menu_remove_action` → `action_remove`

**Comment by bmarty** [comment]
Ok, so I will use `common_verification_cancelled`.

We will be able to remove 
- [ ] `screen_session_verification_cancelled_title`

**Comment by pixlwave** [comment]
> Updating in EXI: https://github.com/vector-im/element-x-ios/pull/1827

Merged on EXI.

**Comment by pixlwave** [comment]
No, we're using all of these except for `screen_login_title`. I'm curious how we've diverged in this way?

**Comment by bmarty** [comment]
Here is a comparative of the 2 apps, Android and iOS:

<img width="946" alt="image" src="https://github.com/vector-im/element-x-android/assets/3940906/6dfce49b-80ef-4d9f-9fe3-c62beef2b53f">

<img width="861" alt="image" src="https://github.com/vector-im/element-x-android/assets/3940906/04bb3568-7a86-4ab3-8466-363a6e66c106">

<img width="854" alt="image" src="https://github.com/vector-im/element-x-android/assets/3940906/e779fad8-c369-4394-84f7-448dffe262c2">

We do not have the same on boarding flow, that's why the strings are not the same.
